README.md

```diff
@@ -97,14 +97,11 @@ The http server is then listening on port 8000.
 ### Services
 #### More
 * Monitoring
-* Matrix Bridges (Telegram, Signal)
+* Matrix Bridges (Signal)
```

Comment:
Why didn't you just do it right now? I mean you just set it up, want are you waiting for?